netlify.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+
+set -euxo pipefail
+
+# Install pnpm
+npm install pnpm serverless --no-progress --no-audit --no-fund --no-save --no-package-lock
+
+# Install dependencies
+./node_modules/.bin/pnpm i --shamefully-hoist
+
+# Build docs website
+cd ./apps/govuk-docs/
+npm run build
+
+# Package functions
+../../node_modules/.bin/sls package
+echo "process.env['MODE'] = 'serverless'; module.exports = require('./dist/server/index.js');" > govuk-docs.js
+zip -rv .serverless/govuk-docs.zip govuk-docs.js
+rm govuk-docs.js
```

Comment:
May I suggest:
```shell
echo "process.env['MODE'] = 'serverless'; module.exports = require('./dist/server/index.js');" | zip -v .serverless/govuk-docs -
```
...in place of the above 3 lines?

Reply:
...actually, sadly the filename comes out as `-` which probably isn't desirable. How about just delete the target file with the `-m` flag instead (of the zip/rm lines):
```shell
zip -mv .serverless/govuk-docs.zip govuk-docs.js
```